src/backend/funmooc/urls.py

```diff
@@ -32,7 +33,15 @@
 urlpatterns += i18n_patterns(
     url(r"^admin/", admin.site.urls),
     url(r"^accounts/", include("django.contrib.auth.urls")),
-    url(r"^", include("cms.urls"))  # NOQA
+    url(
```

Comment:
fun-mooc is a real site not like richie. So we don't want the styleguide if DEBUG is False